src/Components/MinistryTeams/Rides/CreateEventSignupForm/CreateEventSignupForm.scss

```diff
@@ -0,0 +1,83 @@
+/**
+  Template styling.
+ */
+@use 'src/styles/colors';
+@use '~sass-rem' as rem;
+
+.form-creator {
+  border-color: black;
+  border-style: solid;
+  border-radius: 1.5rem;
```

Comment:
maybe we use px and convert using rem.convert(...) here. I used sass-rem to make it easier to translate px to rem.